docs/src/test/scala/docs/http/scaladsl/ServerSentEventsExampleSpec.scala

```diff
@@ -0,0 +1,44 @@
+package docs.http.scaladsl
+
+import akka.NotUsed
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.sse.ServerSentEvent
+import akka.http.scaladsl.server.{ Directives, Route, RoutingSpec }
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.scaladsl.Source
+import docs.CompileOnlySpec
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter.ISO_LOCAL_TIME
```

Comment:
The copy-n-paste-ability would be higher with a few more imports moved into the snippets IMHO.